app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_tag '/register', method: :post do %>
+  <%= label_tag :name %>
+  <%= text_field_tag :name, "#{@user.name}" %>
```

Comment:
References to @user are to pull in previously populated data on redirect for incomplete form.